jwplayer-appletv-web-app/resources/configs/VCyJXbpY/config.json

```diff
@@ -1,11 +1,25 @@
 {
   "playlists": [
-    "K6Sl8yPJ",
-    "BXatQw7p",
-	  "E12RS6r7"
+    {
+      "playlistId": "K6Sl8yPJ",
+      "autoAdvance": true,
```

Comment:
@pablos1 I reverted some changes for now, we can always switch to configuration options per playlist in the future when we need it.